docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -952,3 +952,144 @@ the `${maven.multiModuleProjectDirectory}` will be pointing to the directory fro
+In a multi-module project `quarkus:info` will assume that the current module, in which it is executed, is the main module of the application.
+
+NOTE: The report generated by the `quarkus:info` is not currently including the Quarkus Maven plugin information, however it's planned to be added in the future releases.
+
+Here is an example `info` output for a simple project:
+[source,text,subs=attributes+]
+----
+[aloubyansky@localhost code-with-quarkus]$ mvn quarkus:info
+[INFO] Scanning for projects...
+[INFO] 
+[INFO] ---------------------< org.acme:code-with-quarkus >---------------------
+[INFO] Building code-with-quarkus 1.0.0-SNAPSHOT
+[INFO] --------------------------------[ jar ]---------------------------------
+[INFO] 
+[INFO] --- quarkus-maven-plugin:{quarkus-version}:info (default-cli) @ code-with-quarkus ---
+[WARNING] quarkus:info goal is experimental, its options and output may change in future versions
+[INFO] Quarkus platform BOMs: <1>
+[INFO]   io.quarkus.platform:quarkus-bom:pom:{quarkus-version}
+[INFO]   io.quarkus.platform:quarkus-kogito-bom:pom:{quarkus-version}
+[INFO]   io.quarkus.platform:quarkus-camel-bom:pom:{quarkus-version}
+[INFO] 
+[INFO] Extensions from io.quarkus.platform:quarkus-bom: <2>
+[INFO]   io.quarkus:quarkus-resteasy-reactive
+[INFO] 
+[INFO] Extensions from io.quarkus.platform:quarkus-kogito-bom: <3>
+[INFO]   org.kie.kogito:kogito-quarkus-decisions
+[INFO] 
+[INFO] Extensions from io.quarkus.platform:quarkus-camel-bom: <4>
+[INFO]   org.apache.camel.quarkus:camel-quarkus-rabbitmq
+[INFO] 
+[INFO] Extensions from registry.quarkus.io: <5>
+[INFO]   io.quarkiverse.prettytime:quarkus-prettytime:0.2.1
+----
+
+<1> Quarkus platform BOMs imported in the project (BOMs imported by parent POMs will also be reported)
+<2> Direct Quarkus extension dependencies managed by the `quarkus-bom`
+<3> Direct Quarkus extension dependencies managed by the `quarkus-kogito-bom`
+<4> Direct Quarkus extension dependencies managed by the `quarkus-camel-bom`
+<5> Direct Quarkus extensions dependencies that aren't managed by Quarkus BOMs but found in the Quarkus extension registry
+
+NOTE: `quarkus:info` will also report Quarkus extensions that aren't found in the Quarkus extension registries if those are present among the project dependencies, indicating they have an 'unknown origin'.
+
+[[project-info-misaligned]]
+=== Highlighing misaligned versions
+
+`quarkus:info` will also highlight basic Quarkus dependency version misalignments, in case they are detected. For example, if we modify the project mentioned above by removing the `kogito-quarkus-decisions` extension from the dependencies and adding `2.6.3.Final` version element to the `quarkus-resteasy-reactive` dependency that is managed by the `quarkus-bom` and then run `quarkus:info` again, we'll see something like:
```

Comment:
```suggestion
`quarkus:info` will also highlight basic Quarkus dependency version misalignments, in case they are detected. For example, if we modify the project mentioned above by removing the `kogito-quarkus-decisions` extension from the dependencies and adding a `2.6.3.Final` `<version>` element to the `quarkus-resteasy-reactive` dependency that is managed by the `quarkus-bom` and then run `quarkus:info` again, we'll see something like:
```